src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -112,6 +112,21 @@ const ReportActionItemFragment = (props) => {
                             ? `<email-comment>${htmlContent}</email-comment>`
                             : `<comment>${htmlContent}</comment>`}
                     />
+                ) : (
+                    <Text
+                        selectable={!canUseTouchScreen() || !props.isSmallScreenWidth}
+                        style={[EmojiUtils.containsOnlyEmojis(props.fragment.text) ? styles.onlyEmojisText : undefined, styles.ltr]}
+                    >
+                        {`\u2066${Str.htmlDecode(props.fragment.text)}`}
```

Comment:
@Karim-30 can we also move this to a function? It should prepend `\u2066` to a string

Reply:
Done.